.gitignore

```diff
@@ -9,3 +9,6 @@
 Icon?
 ehthumbs.db
 Thumbs.db
+
+# Android Studio
+.idea/
```

Comment:
We already have a number of IntelliJ files in the Java and Kotlin subfolder .gitignore files. Potentially we can move these items all up to this root .gitignore, but we don't need it to be duplicated.

Reply:
Good point. I'll look into it as a seperate PR.